submission/submission-webapp/src/main/kotlin/ac/uk/ebi/biostd/submission/domain/helpers/FireFilesSource.kt

```diff
@@ -66,6 +64,8 @@ private class SubmissionFireFilesSource(
     override fun getFile(path: String, md5: String?): File? =
         if (md5 == null) fireClient.downloadByPath(subPath.resolve(path).toString())
         else fireClient.downloadByMd5(md5)
+
+    override fun description(): String = "Submission $accNo files"
```

Comment:
Perhaps we could include the version as well?

Reply:
Same as above.